### ISSUE TITLE
about fPlanarVertexRatio 

### ISSUE THREAD

**Issue by Just5D** [cdcseacave/openMVS#43]
Thank you for all these awesome libraries! 
any plan to add "fPlanarVertexRatio " in refineMesh?

**Comment by cdcseacave** [comment]
Indeed, there are plans for it, but I am busy with some other work now. But anybody who wants to contribute is encouraged to do so! Till now all of you were quite shy...

**Comment by Just5D** [comment]
Tying to implement adaptiveMesh in SceneRefineCuda similar already in refine_cpu version, any sugesstion?

**Comment by cdcseacave** [comment]
Closing as stale. The OpenMVS codebase has undergone extensive refactoring and modernisation on the \develop\ branch since this issue was opened. If you believe this is still relevant, please test against the latest \develop\ branch and open a new issue with reproduction steps.
